libraries/botbuilder-dialogs-adaptive-testing/tests/action.test.js

```diff
@@ -9,7 +9,6 @@ const {
     skillConversationIdFactoryKey,
     skillClientKey,
 } = require('botbuilder-dialogs-adaptive');
-
```

Comment:
Why did you removed this line?

Reply:
Probably, I didn't see that it wasn't another import line. I'll restore it.